plugins/module_utils/identity/keycloak/keycloak.py

```diff
@@ -98,6 +101,69 @@ def keycloak_argument_spec():
     )
 
 
+def keycloak_clientsecret_module():
```

Comment:
Hmm, I think this doesn't really belong in this generic file. Could you create a new file in this directory, say `keycloak_clientsecret.py`, that has `keycloak_clientsecret_module` and `keycloak_clientsecret_module_resolve_params`?

Reply:
Done, thanks